src/actions/pause/resumed.js

```diff
@@ -20,13 +20,12 @@ export function resumed() {
   return async ({ dispatch, client, getState }: ThunkArgs) => {
     const why = getPauseReason(getState());
     const wasPausedInEval = inDebuggerEval(why);
+    const wasStepping = isStepping(getState());
 
-    if (!isStepping(getState()) && !wasPausedInEval) {
+    dispatch({ type: "RESUME" });
+
+    if (!wasStepping && !wasPausedInEval) {
```

Comment:
we should evaluate the expressions after the state is updated from the resume dispactch.